### ISSUE TITLE
Revisit commit message conventions in package.json

### ISSUE THREAD

**Issue by TylorS** [motorcyclets/motorcycle#222]
Currently we're using angular style commit messages in our NPM scripts. With #220 there will be almost no viable need for these style of commit messages and they should likely be removed.

**Comment by Frikki** [comment]
Yes. Remove any git commit message convention. Let people write what they deem appropriate. PR review processes will take care of any suggested corrections.
